tools/integration_tests/managed_folders/admin_permissions_test.go

```diff
@@ -60,8 +60,8 @@ func (s *managedFoldersAdminPermission) Setup(t *testing.T) {
 	if s.managedFoldersPermission != "nil" {
 		providePermissionToManagedFolder(bucket, path.Join(testDir, ManagedFolder1), serviceAccount, s.managedFoldersPermission, t)
 		providePermissionToManagedFolder(bucket, path.Join(testDir, ManagedFolder2), serviceAccount, s.managedFoldersPermission, t)
-		// Waiting for 10 seconds as it usually takes 10 seconds for policy changes to propagate.
-		time.Sleep(10 * time.Second)
+		// Waiting for 18 seconds for policy changes to propagate.
```

Comment:
just curious - why 18 and not just 20 ?

Reply:
Thought to keep as less as possible. 15s worked but on louhi one of the OS it failed. That's why increased it to 18s further.

Reply:
let's make it 20s (2s buffer) and write that in a comment that this value is based on experiment.

Reply:
Done